pkg/manifest/manifest_test.go

```diff
@@ -660,23 +676,62 @@ func Test_include(t *testing.T) {
 		},
 		{
 			name:               "incorrect techpreview value is not excluded if techpreview off",
-			includeTechPreview: &falseBol,
+			requiredFeatureSet: utilpointer.String(""),
 			profile:            &defaultClusterProfile,
 			annotations: map[string]interface{}{
 				"include.release.openshift.io/self-managed-high-availability": "true",
 				"release.openshift.io/feature-gate":                           "Other",
 			},
-			expected: fmt.Errorf("unrecognized value release.openshift.io/feature-gate=Other"),
+			expected: fmt.Errorf("unrecognized value \"Other\" in release.openshift.io/feature-gate=Other; known values are: CustomNoUpgrade,Default,IPv6DualStackNoUpgrade,LatencySensitive,TechPreviewNoUpgrade"),
 		},
 		{
 			name:               "incorrect techpreview value is not excluded if techpreview on",
-			includeTechPreview: &trueBol,
+			requiredFeatureSet: utilpointer.String("TechPreviewNoUpgrade"),
 			profile:            &defaultClusterProfile,
 			annotations: map[string]interface{}{
 				"include.release.openshift.io/self-managed-high-availability": "true",
 				"release.openshift.io/feature-gate":                           "Other",
 			},
-			expected: fmt.Errorf("unrecognized value release.openshift.io/feature-gate=Other"),
+			expected: fmt.Errorf("unrecognized value \"Other\" in release.openshift.io/feature-gate=Other; known values are: CustomNoUpgrade,Default,IPv6DualStackNoUpgrade,LatencySensitive,TechPreviewNoUpgrade"),
+		},
+		{
+			name:               "correct techpreview value is excluded if techpreview off using feature-set",
```

Comment:
how does this differ from `correct techpreview value is excluded if techpreview off`?  Ah, that one uses the `release.openshift.io/feature-gate` annotation.  Can we add that qualifier to the existing test-case name `... using feature-gate`?